engine/escalationmanager/db.go

```diff
@@ -82,6 +83,12 @@ func NewDB(ctx context.Context, db *sql.DB, log *alertlog.Store) (*DB, error) {
 			returning ep_step_id, user_id
 		`),
 
+		cleanupMaintenanceMode: p.P(`
```

Comment:
nit: be a little more specific in what this one does in the name

```suggestion
		clearMaintExpiredSvc: p.P(`
```